src/migration/1687956058691-CommentReport.ts

```diff
@@ -0,0 +1,20 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class CommentReport1687956058691 implements MigrationInterface {
+    name = 'CommentReport1687956058691'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`CREATE TABLE "comment_report" ("commentId" text NOT NULL, "userId" character varying(36) NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "reason" character varying(12) NOT NULL, "note" text, CONSTRAINT "PK_cef0f7f862ea6ffd5a99dd640eb" PRIMARY KEY ("commentId", "userId"))`);
```

Comment:
In relation to the initial comment about extending the length.
```suggestion
        await queryRunner.query(`CREATE TABLE "comment_report" ("commentId" text NOT NULL, "userId" character varying(36) NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "reason" character varying(16) NOT NULL, "note" text, CONSTRAINT "PK_cef0f7f862ea6ffd5a99dd640eb" PRIMARY KEY ("commentId", "userId"))`);
```

Reply:
Added `36` as Chris suggested.